schemas/dbt/run-results/v4.json

```diff
@@ -37,12 +37,12 @@
         },
         "dbt_version": {
           "type": "string",
-          "default": "1.2.0a1"
+          "default": "1.4.0a1"
```

Comment:
It looks like only the default and description fields are changing.  Do we really need to commit this?